kover-gradle-plugin/src/main/kotlin/kotlinx/kover/gradle/plugin/dsl/KoverProjectExtension.kt

```diff
@@ -77,6 +77,13 @@ public interface KoverProjectExtension {
      */
     public fun currentProject(block: Action<KoverCurrentProjectVariantsConfig>)
 
+    /**
+     * Instance to configuring of report variants shared by the current project.
+     *
+     * See details in [currentProject].
```

Comment:
Since KDoc doesn't have any syntax for overloads, I'm afraid that `[currentProject]` will link recursively to this property again (it doesn't do this in IDEA currently, but who knows). Maybe just say `See details in [currentProject] function` or `See details in [KoverCurrentProjectVariantsConfig]`?

Reply:
In dokka when you click on the `currentProject` property, the function opens %)

So, it seems to me that this postscript is more for the IDE.